docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1106,19 +1107,89 @@ specific functions.
 
 The methods `register` and `unregister` on `Authenticator` are deprecated in favor of the `Config` system. This means that any custom Authenticator needs to be registered through the yml config:
 ```yaml
-SilverStripe\Security\Authenticator;
-  authenticators:
-    - MyVendor\MyModule\MyAuthenticator
+SilverStripe\Core\Injector\Injector:
```

Comment:
Most of this should move to the actual security docs - it's less of an upgrade concern. The changelog is getting far too long with this stuff all inlined.